src/templates/blocked_ip/list.pug

```diff
@@ -0,0 +1,27 @@
+extends ../components/base.pug
+
+block title
+  title #{title} : Blocked IP List
+
+block content
+  div.columns
+    div.column
+      h3.title.is-3.has-icon
```

Comment:
HTML spec asks that headers not skip levels - i.e. first use h1 then later in the page use h2 and so on without jumping from hN to h(N+x) where x > 1.